openslides/utils/rest_api.py

```diff
@@ -89,6 +90,86 @@ class ViewSet(PermissionMixin, _ViewSet):
     pass
 
 
+class RESTElement(metaclass=RegisterSubclasses):
+    """
+    Element of our REST api.
+
+    Each element bundels a viewset, a route and optionally a model together.
+    """
+
+    viewset = None
+    """
+    A django REST framework viewset class used in the router.
+    """
+
+    router_prefix = None
```

Comment:
This is also used as router's base_name which is not obvious when reading the docstring.